apps/prairielearn/assets/scripts/lib/saveButtonEnabling.ts

```diff
@@ -10,7 +10,15 @@ export function saveButtonEnabling(form: HTMLFormElement, saveButton: HTMLButton
   // Create a record to store the default values of select elements. This is because we do not have access to the default value of a select element like we do for an input element.
   const selectDefaultValues: Record<string, any> = {};
   form.querySelectorAll('select').forEach((element) => {
-    selectDefaultValues[element.id] = (element as HTMLSelectElement).value;
+    if (element.multiple) {
+      selectDefaultValues[element.id] = Array.from(
+        (element as HTMLSelectElement).selectedOptions,
+      ).map((option) => option.value);
+    } else {
+      if (element.id && element.id !== '') {
+        selectDefaultValues[element.id] = (element as HTMLSelectElement).value;
+      }
+    }
```

Comment:
The casts here shouldn't be necessary; the types for `querySelectorAll` are sophisticated enough that `element` is known to be an `HTMLSelectElement` here.

```suggestion
    if (element.multiple) {
      selectDefaultValues[element.id] = Array.from(element.selectedOptions).map(
        (option) => option.value
      );
    } else {
      if (element.id && element.id !== '') {
        selectDefaultValues[element.id] = element.value;
      }
    }
```